### PR TITLE
UIDATIMP-61: Implement data-import settings/file extension/ details page

### PR DESCRIPTION
**Pull request by oleksii-popov**
### Purpose

Implement data-import settings/file extension/ details page

### Screenshots
screenshot at feb 25 20-16-53
screenshot at feb 25 20-16-40

### CODE REVIEW

File: src/settings/FileExtensions/ViewFileExtension.js [hunk 1]
```diff
@@ -42,6 +49,12 @@ export class ViewFileExtension extends Component {
     onClose: PropTypes.func.isRequired,
   };
 
+  constructor(props) {
+    super(props);
+
+    this.cViewMetaData = props.stripes.connect(ViewMetaData);
```
Comment (OleksandrYehorovEPAM) [inline src/settings/FileExtensions/ViewFileExtension.js]:
```suggestion
    const { stripes } = this.props;

    this.connectedViewMetaData = stripes.connect(ViewMetaData);
```
Can you rewrite it like this for consistency. Like in Home and DataImportSettings components

File: src/settings/FileExtensions/ViewFileExtension.js [hunk 2]
```diff
@@ -105,6 +153,64 @@ export class ViewFileExtension extends Component {
             >
               {record.extension}
             </Headline>
+
+            {record.metadata && (
+              <Row>
+                <Col xs={12}>
+                  <this.connectedViewMetaData metadata={record.metadata} />
```
Comment (VictorSoroka1) [inline src/settings/FileExtensions/ViewFileExtension.js]: ViewMetaData in the current state does not suit well for the case when the record should be updated by System.

File: src/components/EndOfRecord/EndOfRecord.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+import {
+  Layout,
+  Icon,
+} from '@folio/stripes/components';
+
+import css from './EndOfRecord.css';
+
+export const EndOfRecord = () => (
+  <Layout className={classnames(css.endOfRecord)}>
```
[the remark this answers is not in the document]
Comment (oleksii-popov) [reply src/components/EndOfRecord/EndOfRecord.js]: Thanks

File: src/settings/FileExtensions/ViewFileExtension.js [hunk 4]
```diff
@@ -8,12 +8,19 @@ import { FormattedMessage } from 'react-intl';
...
+  KeyValue,
+  Icon,
+  Button,
+  PaneMenu,
 } from '@folio/stripes/components';
 import { TitleManager } from '@folio/stripes/core';
 
 import { Preloader } from '../../components/Preloader';
+import { EndOfRecord } from '../../components/EndOfRecord';
+import { ViewMetaData } from '@folio/stripes/smart-components';
```
[the remark this answers is not in the document]
Comment (oleksii-popov) [reply src/settings/FileExtensions/ViewFileExtension.js]: Yes, updated

File: src/settings/FileExtensions/ViewFileExtension.js (hunk 1 above)
Comment (oleksii-popov) [reply src/settings/FileExtensions/ViewFileExtension.js]: Refactored

**Pull request merged**